util/topsql/reporter/reporter.go

```diff
@@ -133,6 +134,18 @@ func NewRemoteTopSQLReporter(client ReportClient) *RemoteTopSQLReporter {
 	return tsr
 }
 
+var (
+	ignoreRegisterSQLCnt          = metrics.TopSQLIgnoredCounter.WithLabelValues("register-sql")
+	ignoreRegisterPlanCnt         = metrics.TopSQLIgnoredCounter.WithLabelValues("register-plan")
+	ignoreCollectCPUTimeCnt       = metrics.TopSQLIgnoredCounter.WithLabelValues("collect-data")
+	ignoreReportDataCnt           = metrics.TopSQLIgnoredCounter.WithLabelValues("report-data")
+	reportDurationSuccHistogram   = metrics.TopSQLReportDurationHistogram.WithLabelValues("OK")
+	reportDurationFailedHistogram = metrics.TopSQLReportDurationHistogram.WithLabelValues("Error")
```

Comment:
If I remember correctly there are constant labels for OK and Error, maybe we can reuse them?